### PR TITLE
Blocked: Footer updates

### PR DESCRIPTION
**Pull request by wilwade**
Updates to the footer to match the new styles

# Solution

- Add light/dark options to the footer
- Remove the terms
- Added option to override the privacy policy link

## Screenshots (optional):

(Backgrounds are added outside of the component)

image

image

image

image

### CODE REVIEW

**Review by claireolmstead — APPROVED** [review]
Tested and read code. Looks great. Added a small commit putting a max width on the footer in XL screens.
Demo:
Screenshot 2024-11-07 at 4 16 23 PM

File: src/lib/features/Footer.svelte [hunk 1]
```diff
@@ -1,20 +1,39 @@
...
 
-<footer class="sm md:md flex flex-col items-center gap-f16 border-t border-black py-f48 md:gap-f24">
-  <div class="flex gap-f40">
-    <a href="/privacy" class="underline transition duration-[0.3s] hover:text-brightBlue">Privacy Policy</a>
-    <a href="/terms" class="underline transition duration-[0.3s] hover:text-brightBlue">Terms of Conditions</a>
-  </div>
+<footer
+  class={cn(
+    `sm md:md flex flex-col items-center gap-f16 border-t-[2px] border-current text-${fgColor} mx-auto my-0 max-w-screen-xl py-f48 md:gap-f48`
+  )}
```
Comment (JoeCap08055) [inline src/lib/features/Footer.svelte:18]: Watch out--building TW class names with dynamic strings (`text-${fgColor}`) can be problematic.
Also, if you're not merging/concatenating classes, I don't think you need the `cn(...)` method; you can just say `class="..."`.

Or does the cn(...) function somehow overcome the gotchas that dynamic TW class names presents?

Anyway, see PR #50 for an alternate approach to this styling issue.

Comment (JoeCap08055) [reply src/lib/features/Footer.svelte:18]: I think TW recommends something more like the following:
```svelte
<footer class="... { fgColor === 'light' ? 'text-black' : 'text-white'}"/>
```

Comment (JoeCap08055) [reply src/lib/features/Footer.svelte:18]: ☝🏻 reason being that the TW parses is a very simple string-matcher on class names; it doesn't recognize `text-${fgColor}` as a TW class, but it does fine with `'text-white' : 'text-black'` (it compiles both styles)

**Pull request merged**